rspack.config.ts

```diff
@@ -331,7 +306,7 @@ const appConfig: rspack.Configuration = {
       'process.env.NODE_ENV': JSON.stringify(env.NODE_ENV),
       'process.env.IS_ACCEPTANCE_TEST': JSON.stringify(env.IS_ACCEPTANCE_TEST),
       'process.env.DEPLOY_PREVIEW_CONFIG': JSON.stringify(env.DEPLOY_PREVIEW_CONFIG),
-      'process.env.EXPERIMENTAL_SPA': JSON.stringify(env.EXPERIMENTAL_SPA),
```

Comment:
env.EXPERIMENTAL_SPA is undefined, so it cause infinite reload because of the following code https://github.com/getsentry/sentry/blob/master/static/app/api.tsx#L172-L176